eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -73,7 +74,16 @@ jobs:
         value: ${{ parameters.Location }}
 
     strategy:
-      matrix: ${{ parameters.Matrix }}
+      matrix: 
+        ${{ insert }}: ${{ parameters.Matrix }}
+        ${{ if eq(parameters.TestCanary, 'true') }}:          
+          Linux - Java 8 (AzureCloudCanary):            
+            OSVmImage: 'ubuntu-18.04'            
+            JavaTestVersion: '1.8'            
+            DisplayName: 'Linux - Java 8'            
+            SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)            
+            Location: centraluseuap            
```

Comment:
Can you set this as a param with default value "centraluseuap"

Reply:
Done!